drivers/gpu/drm/vc4/vc4_plane.c

```diff
@@ -1621,6 +1624,8 @@ struct drm_plane *vc4_plane_init(struct drm_device *dev,
 					  DRM_COLOR_YCBCR_BT709,
 					  DRM_COLOR_YCBCR_LIMITED_RANGE);
 
+	drm_plane_create_chroma_siting_properties(plane, 0, 0);
```

Comment:
Do the default properties get set based on __drm_atomic_helper_plane_state_reset?

Ah, that's not been merged into the tree yet. I thought I'd backported it.
https://cgit.freedesktop.org/drm/drm-misc/log/drivers/gpu/drm/drm_atomic_state_helper.c https://cgit.freedesktop.org/drm/drm-misc/commit/drivers/gpu/drm/drm_atomic_state_helper.c?id=1a7998dab5dd3d11bada7e3921781922082e7fe6

Ah, was part of #4846, and not backported since Maxime took it over and extended it. I ought to do that.

Reply:
#4846 has been updated and merged. It'd be worth adding the restore to ```__drm_atomic_helper_plane_state_reset``` as part of "drm: Add chroma siting properties" to ensure the default values are loaded. It'll make no difference in this case as the default is 0,0, so can come later.